clients/admin-ui/src/features/common/fides-types.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Common Fides types adapted from https://github.com/ethyca/fideslang/blob/main/src/fideslang/models.py
```

Comment:
I wish there were a way to share types or autogenerate types across the front and back-ends. Some sort of ORM layer would be amazing. Probably overkill at this point though :) For now keeping a link in the comment is 👌, good call.

Reply:
totally agree, I was just thinking that as I was making these. since the backend does use strongly typed pydantic, it should be doable... one day :)